services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
Code Style issue: You should use `get_user_model()` instead of `User` as per the checklist."Good example:\nget_user_model().objects.create_user(\n    username=username, password=password\n)\n\nBad example:\nUser.objects.create_user(\n    username=username, password=password\n)"